python/instrumentation/openinference-instrumentation-langchain/src/openinference/instrumentation/langchain/_tracer.py

```diff
@@ -484,44 +484,46 @@ def _get_tool_call(tool_call: Optional[Mapping[str, Any]]) -> Iterator[Tuple[str
 
 
 @stop_on_exception
-def _prompt_template(run: Run) -> Iterator[Tuple[str, Any]]:
-    """
-    A best-effort attempt to locate the PromptTemplate object among the
-    keyword arguments of a serialized object, e.g. an LLMChain object.
-    """
-    serialized: Optional[Mapping[str, Any]] = run.serialized
-    if not serialized:
-        return
-    assert hasattr(serialized, "get"), f"expected Mapping, found {type(serialized)}"
-    if not (kwargs := serialized.get("kwargs")):
+def _parse_prompt_template(
```

Comment:
might be helpful to have tests for this. Seems like a good spot since it has a fair number of permutations.